scripts/replace-user.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+"use strict";
+const replace = require('replace-in-file');
+/*
+	usage: node replace-user <hash value>
+	run in augur-node root dir
+*/
+
+if (process.argv.length < 3) throw new Error('Pass in user hash as arguments');
+
+process.argv.slice(2).map(function (user_hash) {
```

Comment:
Also, the structure of this program is to iterate on all arguments, and i don't think it makes sense to run through this map twice. If you want to just leave a lot of structure intact, you could just 

`slice(2,3)` and make sure the process.argv.length == 3